packages/core/admin/server/src/utils/index.js

```diff
@@ -1,9 +1,7 @@
-'use strict';
+import {} from '@strapi/types';
```

Comment:
Why is this needed?

Reply:
@innerdvations wanted to ask about this.
@strpai/types injects the global strapi variable type definition, and as this file is using the strapi global, the only way I could find was to import strapi types like this

Reply:
You could always do `import '@strapi/types'` instead.

But ideally, we should finally just find a way to remove the strapi global from everywhere we can in V5